### PR TITLE
 feature:add start process for readme 

### PR DESCRIPTION
**Pull request by OctopusHu0105**
feature:add start process for readme

### CODE REVIEW

**Comment by OctopusHu0105** [comment]
@shootao @withdra  PTAL

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+
+
+## 开发管理
+
+# 分支管理
+
+
+提交代码分为两种 feature 和 bugfix。
+feature 提交实例:
+git checnkout -b feature/add_printf_hello_for_hello_word_examples
```
Comment (withdra) [inline README.md]: typo:  checnkout  --> checkout

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+git commit -m "feature: add printf hello for hello word examples"
+
+git push
+
+bugfix 提交实例：
+git checnkout -b bugfix/fix_printf_crash_in_hello_word_examples
```
Comment (withdra) [inline README.md]: typo: checnkout --> checkout

**Comment by withdra** [comment]
@OctopusHu0105 Left some comments, others LGTM.

**Pull request merged**